### PR TITLE
Add: Comments and Styling

### PR DESCRIPTION
**Pull request by yozzyurubuza**
Description

Added comments to explain the functions and flow of the code in better detail
Added some css styling for font size and video-item size
Deleted some unnecessary files and some comments
Added Error-handling on fetching data from youtube API.

Used !important tag in css temporarily to force implementation of some code, will fix this code in the future.

### CODE REVIEW

File: 11-test-react-mastery/src/components/App.css [hunk 1]
```diff
@@ -1,5 +1,7 @@
 body {
   background-image: url("https://i.ibb.co/rcyn0rs/ro-background.jpg") !important;
+  object-fit: fill !important;
```
Comment (jcgalbano) [inline 11-test-react-mastery/src/components/App.css:3]: https://stackoverflow.com/questions/3706819/what-are-the-implications-of-using-important-in-css

Read this and refactor accordingly, this is a good time to research why you need to apply !important

**Review by jcgalbano — CHANGES_REQUESTED** [review]
looks good just an opportunity to learn and refactor

File: 11-test-react-mastery/src/components/App.css (hunk 1 above)
Comment (yozzyurubuza) [reply 11-test-react-mastery/src/components/App.css]: I was able to remove the important tags, but is this okay? 
The codes from the 3rd party library were crossed out, which is similar when implementing !important.

Comment (jcgalbano) [reply 11-test-react-mastery/src/components/App.css]: Code*, can you link and explain to me what you did differently in this PR? Explore the UI of the "Files changed" tab and comment accordingly.

**Comment by jcgalbano** [comment]
@yozzyurubuza Also try to distinguish your commit messages. I see that you have 2 Edit width and 2 Edit row width. Maybe u can specify which one like Edit MyComponent width. That's more intuitive i think.

File: 11-test-react-mastery/src/css/App.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.custom {
```
Comment (yozzyurubuza) [inline 11-test-react-mastery/src/css/App.css:1]: Created custom class in css for background

Comment (jcgalbano) [reply 11-test-react-mastery/src/css/App.css:1]: just changed this to a more descriptive class name and we're good

Comment (yozzyurubuza) [reply 11-test-react-mastery/src/css/App.css]: Done, used a descriptive class name.

**Review by jcgalbano — APPROVED** [review]
good job, looks good can merge

**Pull request merged**